<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { background-color: black; color: white; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="user_label">
     <property name="text">
      <string>Usu√°rio:</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="user_entry">
     <property name="styleSheet">
      <string notr="true">background-color: black; color: white;</string>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="password_label">
     <property name="text">
      <string>Senha:</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="password_entry">
     <property name="styleSheet">
      <string notr="true">background-color: black; color: white;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="login_button">
     <property name="text">
      <string>Login</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: green; color: white;</string>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
